package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-mbaas-api",
-  "version": "5.13.1",
+  "version": "5.13.2",
```

Comment:
@grdryn absolutely, we'd appreciate a patch release whenever it's feasible for you folks 👍  We're not too concerned about templates here 😄